### DISCUSSION TITLE
How to add a footer to a desktop app layout

### DISCUSSION THREAD

**Discussion by clstaudt** [flet-dev/flet#300] [Q&A]
![Screenshot 2022-09-09 at 16 52 07](https://user-images.githubusercontent.com/875194/189379416-7ddb670d-9715-4ed3-9436-4f077c11942d.png)

A  typical layout pattern for a desktop app is having a footer at the bottom of the application window.

Looking for a way to implement this with `flet`.

**Comment by FeodorFitsner** [comment]
Here you go:

```python
import flet
from flet import Column, Container, Page, Row, Text

def main(page: Page):

    main_content = Column(scroll="auto")

    for i in range(100):
        main_content.controls.append(Text(f"Line {i}"))

    page.padding = 0
    page.spacing = 0
    page.horizontal_alignment = "stretch"
    page.add(
        Container(main_content, padding=10, expand=True),
        Row([Container(Text("Footer"), bgcolor="yellow", padding=5, expand=True)]),
    )

flet.app(target=main)
```

![image](https://user-images.githubusercontent.com/5041459/189494493-afc560d4-1d27-453d-bac1-626bc28f27b3.png)
